src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -231,9 +243,14 @@ func configureSystemTray() {
 	}
 	systray.SetIcon(icon)
 	systray.SetTooltip("Lantern")
-	quit := systray.AddMenuItem("Quit", "Quit Lantern")
+	show := systray.AddMenuItem(i18n.T("TRAY_SHOW_LANTERN"), i18n.T("SHOW"))
+	quit := systray.AddMenuItem(i18n.T("TRAY_QUIT"), i18n.T("QUIT"))
 	go func() {
-		<-quit.ClickedCh
-		os.Exit(0)
+		select {
```

Comment:
This needs to happen inside a loop, otherwise only one command will ever be executed!